Source/Blazorise/Components/DatePicker/DatePicker.razor.cs

```diff
@@ -381,7 +379,8 @@ private object GetLocalizationObject()
                         Localizer["December"]
                     }
                 },
-                amPM = new[] { Localizer["AM"], Localizer["PM"] }
+                amPM = new[] { Localizer["AM"], Localizer["PM"] },
+                RangeSeparator = RangeSeparator ?? Localizer["RangeSeparator"],
```

Comment:
Use `CurrentRangeSeparator` helper? Which also has a slightly different logic, it fallbacks to `" to "` if everything null, which probably will never happen anyway